js/components/Talent.js

```diff
@@ -64,31 +64,46 @@ class BudgetTalent extends HTMLElement {
           width: 100%;
           text-align: center;
           position: relative;
+          top: 25px;
```

Comment:
If we use `top: 25px;` we can't use margins properly because they will collapse into the top setting. 

Instead of using `top: 25px;` use something like `margin-bottom: 25px;`. This goes for all places where we use top.